src/drawers/canvasDrawer.ts

```diff
@@ -108,6 +110,20 @@ export function getStrokeWidth(style: Record<string, any>) {
   return style["stroke-width"] != null ? parseFloat(style["stroke-width"]) : 1;
 }
 
+export function getStrokeDashArray(style: Record<string, any>): number[] {
+  const rawValue: string = style["stroke-dasharray"];
+  if (rawValue != null) {
+    try {
+      const cleaned = rawValue.replace(/, /g, ",").replace(/ /g, ",");
+      return cleaned.split(",").map((x: string) => parseInt(x, 10));
```

Comment:
I think you can use regexp's in `string.split`, so instead of doing a replace then split i think you could just do `rawValue.split(/[ ,]+/)`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Parameters